modules/ROOT/examples/live-demos/comments-callback/example.js

```diff
@@ -169,15 +169,40 @@ const tinycomments_lookup = ({ conversationUid }, done, fail) => {
     });
 };
 
+const tinycomments_fetch = (conversationUids, done, fail) => {
+  fetch(`https://api.example/conversations/${conversationUids}`, {
```

Comment:
This is not how we expect the api to be used. The `conversationUids` is an array and can't be put in the url. The `conversationUids` array is already used in the body.

```suggestion
  fetch(`https://api.example/conversations`, {
```